samples/glean/metrics.yaml

```diff
@@ -8,6 +8,9 @@
 
 $schema: moz://mozilla.org/schemas/glean/metrics/1-0-0
 
+no_lint:
```

Comment:
nit: since this is a sample file, potentially some file people will copy paste, would you mind adding a comment here explaining that this should *not* be done in production, unless needed, or, even better, change the sample to match our style? ;)

Reply:
Sure.  I've updated the names of the metrics.  This, of course, will cause a discontinuity in the data collected, but I suppose for `sample-glean` it doesn't matter?

Reply:
Yes, we don't really care about this data :)